packages/react/popper/src/Popper.tsx

```diff
@@ -246,6 +255,8 @@ const PopperContent = React.forwardRef<PopperContentElement, PopperContentProps>
         animation: !isPlaced ? 'none' : undefined,
         // hide the content if using the hide middleware and should be hidden
         opacity: middlewareData.hide?.referenceHidden ? 0 : undefined,
+        maxHeight: 'inherit',
+        overflow: 'auto',
```

Comment:
Should consumers of the library be responsible for setting this? It's being added as an inline style with highest specificity.